### ISSUE TITLE
Items disappears when scrolling with SliverQuiltedGridDelegate

### ISSUE THREAD

**Issue by berkaymazlumlar** [letsar/flutter_staggered_grid_view#256]
I added video about this issue. I realized that it's only happening when childCount is more than 25.

https://user-images.githubusercontent.com/54273954/174033196-48d8e2cf-0142-47c4-bb22-4b6a2251e051.mp4

Here is my code: 

```
class MyApp extends StatelessWidget {
  @override
  Widget build(BuildContext context) {
    return MaterialApp(
      title: 'Material App',
      home: Scaffold(
        appBar: AppBar(
          title: Text('Material App Bar'),
        ),
        body: Center(
          child: GridView.custom(
            gridDelegate: SliverQuiltedGridDelegate(
              crossAxisCount: 9,
              repeatPattern: QuiltedGridRepeatPattern.same,
              pattern: const [
                QuiltedGridTile(4, 6),
                QuiltedGridTile(4, 3),
                QuiltedGridTile(4, 3),
                QuiltedGridTile(4, 3),
                QuiltedGridTile(4, 3),
              ],
            ),
            childrenDelegate: SliverChildBuilderDelegate(
              (context, index) => Container(
                color: index % 2 == 0 ? Colors.yellow : Colors.blue,
              ),
              childCount: 30,
            ),
          ),
        ),
      ),
    );
  }
}
```

**Comment by letsar** [comment]
Related to #232 . It will be fixed today. Sorry for the inconvenience.

**Comment by letsar** [comment]
Fixed in 0.6.2.
